### DISCUSSION TITLE
Improvement in report api

### DISCUSSION THREAD

**Discussion by Sourabh-jana** [amzn/ads-advanced-tools-docs#218] [Ideas]
I think there should be a field
 isEmpty | haveData in 
GET
/reporting/reports/{reportId}

so that api user could know that the report generated by the provided params have report data or not
if there is no data in generated report then it is obvious to not to call GET report data api after getting this information in get status response.

**Comment by kdwivedi13** [comment]
Hi Saurabh,

Thanks for the suggestion! I'll share this with the product team and keep you updated on any progress. We value your input's and appreciate you taking the time to share your thoughts.

Thanks and Regards,
Kapil Dwivedi
